src/main/java/server/Server.java

```diff
@@ -0,0 +1,30 @@
+package main.java.server;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Server {
+
+    private static final int PORT = 8080;
+
+    public static void main(String[] args) {
+        try {
+            ServerSocket serverSocket = new ServerSocket(PORT);
+            Socket socket = serverSocket.accept(); // block
+            try {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Extract this nested try block into a separate method. ([squid:S1141](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1141))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=3&issues=AWQTV3H7_bEzQ-kUL94G&open=AWQTV3H7_bEzQ-kUL94G)

[ISSUE_KEY]: AWQTV3H7_bEzQ-kUL94G